src/amo/components/LandingPage.js

```diff
@@ -77,15 +79,25 @@ export class LandingPageBase extends React.Component {
       return contentForTypes[addonType];
     }
 
-    throw new Error(`No LandingPage content for addonType: ${addonType}`);
+    throw new NoAddonTypeError(
+      `No LandingPage content for addonType: ${addonType}`);
   }
 
   render() {
     const {
       addonType, featuredAddons, highlyRatedAddons, popularAddons,
     } = this.props;
 
-    const html = this.contentForType(addonType);
+    let html;
+    try {
+      html = this.contentForType(addonType);
+    } catch (err) {
+      if (err instanceof NoAddonTypeError) {
```

Comment:
Heh, I actually had it, tested it, then took it out as I thought it wouldn't be needed. Fair enough, will restore 😄